### PR TITLE
Fix cleanup

### PR DESCRIPTION
**Pull request by UjjwalAyyangar**
* Fixes popper cleanup 
* Fixes the bad argument bug in `popper init --existing`
* Adds a `popper reset` command for completely resetting a popper repository.

### CODE REVIEW

File: cli/popper/commands/cmd_cleanup.py [hunk 1]
```diff
@@ -15,26 +14,34 @@ def cli(ctx):
     """Synchronize your pipelines and popper.yml file if any pipeline or stage
     has been deleted.
     """
-    pipeline_dir = os.path.join(pu.get_project_root(), 'pipelines')
+    #pipeline_dir = os.path.join(pu.get_project_root(), 'pipelines')
```
Comment (ivotron) [inline cli/popper/commands/cmd_cleanup.py]: instead of commenting, we can remove it

File: cli/popper/commands/cmd_init.py [hunk 2]
```diff
@@ -30,8 +30,8 @@
 @click.option(
     '--infer-stages',
     help=('Infers the name of the stages from the existing bash scripts'
-          'on the folder while creating a new pipeline from the existing'
-          'folder.'),
+          ' on the folder while creating a new pipeline from the existing'
+          ' folder.'),
```
Comment (ivotron) [inline cli/popper/commands/cmd_init.py]: maybe we could reword this to:

```
Infers the stages of a pipeline from bash script file names.
Used in conjunction with the --existing flag.
```

then user can read the help for `init` and for `--existing`

**Comment by UjjwalAyyangar** [comment]
@ivotron - Requested changes have been made.

File: ci/test-reset.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (ivotron) [inline ci/test-reset.sh]: this stages needs to be added to `.popper.yml`

**Comment by UjjwalAyyangar** [comment]
Some more changes to be made. I will notify when they are completed.

**Comment by ivotron** [comment]
thanks a lot!
